content/docs/user-guide/experiment-management/visualizing-plots.md

```diff
@@ -268,6 +268,10 @@ view.
 
 ![VS Code Comparison](/img/dvclive-vscode-compare.png)
 
+The extension can also make plots debugging easier.
+
+https://youtu.be/XH0v3x7lBDg?si=6YheW-ocPU9MMoIq
```

Comment:
We don't really mention plots debugging anywhere that I could find in the docs and I eventually chose this spot, but maybe there are other places?

Reply:
This seems pretty hidden to me. Could we either put it under the other VS Code video in this page or create a new section for debugging plots?

Reply:
Good point! I've updated the video to be under the defining plots video: 

![Screenshot 2024-02-02 at 2 51 00 PM](https://github.com/iterative/dvc.org/assets/43496356/7d7dba1a-969b-4b8a-a6f8-c5ce41091c64)